sdks/python/apache_beam/pipeline_test.py

```diff
@@ -390,6 +391,8 @@ def process(self, element):
     assert_that(pcoll, equal_to([11, 12]))
     pipeline.run()
 
+  @unittest.skipIf(sys.version_info[0] == 3, 'This test still needs to be '
```

Comment:
Do we have a JIRA issue to clean up things like this? Should we tag code with proper TODOs?

Reply:
Yes, I was cucrrently planning to leave the module specific task open until these skips are resolved.